sw/host/opentitanlib/src/transport/ultradebug/mod.rs

```diff
@@ -69,8 +70,6 @@ impl Ultradebug {
         let mut mpsse_b = self.mpsse_b.borrow_mut();
         if mpsse_b.is_none() {
             let device = self.from_interface(ftdi::Interface::B)?;
-            // Read and write timeouts:
-            device.set_timeouts(5000, 5000);
```

Comment:
Does ftdi-rs not provide this setting?

Reply:
No, https://github.com/tanriol/ftdi-rs/blob/master/src/lib.rs#L425

do you think that will be an issue?

Reply:
These are the defaults in libftdi anyway so it should be a no-op:

https://github.com/lipro/libftdi/blob/9da4394efeafa2fcfdc809a9bec3219c0a95e77a/src/ftdi.c#L93

Reply:
Not really, but if we can't keep this functionality equivalent then we need people to confirm that this is okay. @jesultra

Reply:
If we're just explicit setting the default then I think it'll be good. I still prefer to have a confirmation that the code still works given that we don't have ultradebug to test it on.

Reply:
I agree it would be good to get confirmation it works on UltraDebug.

It would also be good to get confirmation that UltraDebug is still used for OpenTitan since it's a little trickier to make code changes around this board.